scripts/create-admin.bash

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# Add a bash script that:
+#
+# * downloads latest release
+# * installs it with `dpkg -i` or `rpm -i`
+# * creates the first admin user with `firezone-ctl create-or-reset-admin ${admin_email}`
+#
+# Prompt for admin email should be along the lines of:
+#
+# `Enter the administrator email you'd like to use for logging into this Firezone instance:`
+#
+# Sanity check by making sure input has a `@`
+#
+# The `create-or-reset-admin` command will need to be updated to optionally accept an email argument.
+
+# * prompts user for admin email
+function promptEmail() {
+  echo $1
+  read adminEmail
+  if [[ $adminEmail != *[@]* ]]
+  then
+    promptEmail "please provide a valid email"
+  else
+    echo Administrator username will be: $adminEmail
+  fi
+}
+
+# * checks to ensure wireguard is available (i.e. wireguard net devices can be added) (maybe there's a /sys file?)
+function wireguardCheck() {
+	if which wg > /dev/null; then
```

Comment:
`wg` is part of the wireguard tools package, not wireguard itself. We bundle wireguard-tools ourselves and so don't rely on `wg` being present.

Maybe check the exit status of `modprobe wireguard` instead? Or something other way to see if the kernel module is available?

Reply:
wireguard is a pre-existing kernal module ... is there any docs or links I can review to understand this?

Reply:
Pre 5.6 it was a separate kernel module so you had to build it with DKMS or something. After 5.6 it's mainlined into the master Linux tree.

For post 5.6 kernels, you can assume it's available. For pre 5.6 kernels, you need to check it's there. Specifically, this command needs to succeed, which will be called later by the installer:

```
# ip link add dev wg0 type wireguard
```

So if there is a way to dry-run that command, that would be ideal.

https://arstechnica.com/gadgets/2020/01/linus-torvalds-pulled-wireguard-vpn-into-the-5-6-kernel-source-tree/

Reply:
```
root@demo:~# cat /sys/module/wireguard/version
1.0.0
```

Reply:
and if the OS doesn't have anything like this image I'm running:

over here I see 🦗 's
```root@ip-172-31-5-138:/home/admin# find /sys -name "*wireguard*"```

what should be the steps if those files are not present or wireguard is not available?

Reply:
Just print out 

```
Error! WireGuard not detected. Please upgrade your kernel to at least 5.6 or install the WireGuard kernel module.
See more at https://www.wireguard.com/install/
```

Reply:
is there a special trick to get access to a centos ec2?
<img width="1045" alt="Screen Shot 2022-02-19 at 8 55 51 PM" src="https://user-images.githubusercontent.com/611899/154827099-155346e6-bc28-48b3-bd02-73e98be471cc.png">
<img width="1006" alt="Screen Shot 2022-02-19 at 8 58 14 PM" src="https://user-images.githubusercontent.com/611899/154827101-c0fa535a-a692-4677-9928-9745a6be6bd3.png">

https://stackoverflow.com/questions/33991816/ec2-ssh-permission-denied-publickey-gssapi-keyex-gssapi-with-mic

my goal is to see what `hostnamect` looks like on an ARM redhat-esque box but what a 🐰 🕳️ ... 
or ideally run that on all of the images you have so I can build a lookup table in this bash script 🤔 

I'll nuke that nano for now since I can't even reach it 🤦

Reply:
@copremesis Try using `ec2-user` as the username

Reply:
@jamilbk 
<img width="1006" alt="Screen Shot 2022-02-19 at 8 58 14 PM" src="https://user-images.githubusercontent.com/611899/155007883-16c71f9a-3c19-4ea4-9188-06e49b7da54b.png">

Reply:
yeah so Debian works fine it's the oddity of the cento's that's driving me nuts ... looking at your packer file it appears that default user would be centos but oddly can't seem to _guess_ the correct username